pyproject.toml

```diff
@@ -55,7 +55,13 @@ dependencies = [
 
 [project.optional-dependencies]
 dev = [
+    "black",
+    "flake8",
+    "freezegun",
+    "isort",
```

Comment:
I thought we were going with ruff for formatting and linting after @eevelweezel last PR.

Reply:
Not sure, I was getting a merge conflict when I pulled in the main branch and tried to resolve it as best I could (though I just branched off of main today). Did I merge wrong?

Reply:
Upstream or origin 🤔

Reply:
I thought I pulled main from chipy.org official project (and then pushed up that main to my main branch in my local repo). Something must have gotten messed up with that. Anyway, the diff in the merge looks like it now only includes my changes.